gemrb/core/Strings/CString.h

```diff
@@ -122,12 +122,14 @@ class FixedSizeString {
 		return *this;
 	}
 	
-	int SNPrintF(const char* format, ...) noexcept {
-		va_list args;
-		va_start(args, format);
-		int ret = vsnprintf(str, sizeof(str), format, args);
-		va_end(args);
-		return ret;
+	template<typename ...ARGS>
+	bool Format(const char* format, ARGS&& ...args) noexcept {
+		auto ret = fmt::format_to_n(begin(), LEN + 1, format, std::forward<ARGS>(args)...);
+		if (ret.size > LEN) {
```

Comment:
should we also be doing it for 0?

Reply:
From what you changed, yeah. It's safe to leave it as-is and if we ever need to detect useless calls somewhere, that can be added.